src/components/item-filter/index.tsx

```diff
@@ -1,60 +1,27 @@
-import React, { useState, useEffect } from 'react';
+import { IRenderTrackParams } from 'react-range/lib/types';
+import { Range } from 'react-range';
 import clsx from 'clsx';
 import NiceSelect from '@ui/nice-select';
-import { Range } from 'react-range';
-import { IRenderTrackParams } from 'react-range/lib/types';
-import {
-  selectCoastedItem,
-  selectFilteredItems,
-} from '../../features/leda-nft/store/leda-nft.slice';
-import useAppSelector from '../../store/hooks/useAppSelector';
-import SliderTrack from '../ui/input-range/slider-track';
+import React, { useState } from 'react';
 import SliderThumb from '../ui/input-range/slider-thumb';
-import { Props, FilterType } from '../../types/item-filter-types';
+import SliderTrack from '../ui/input-range/slider-track';
 import useAppDispatch from '../../store/hooks/useAppDispatch';
-import { selectNFTsMarketplace } from '../../features/marketplace/store/marketplace.slice';
-import { findMarketplace } from '../../features/marketplace/store/marketplace.actions';
-
-const cheapestStr = 'cheapest';
-const expensiveStr = 'expensive';
-
-const ItemFilter = ({ setNfts }: Props) => {
-  /* const dispatch = useAppDispatch();
-  const { NFTs, isLoading } = useAppSelector(selectNFTsMarketplace);
-
-  useEffect(() => {
-    dispatch(findMarketplace());
-  }, [dispatch]); */
-
-  const cheapest: number = useAppSelector((state) => selectCoastedItem(state, cheapestStr));
-  const mostExpensive: number = useAppSelector((state) => selectCoastedItem(state, expensiveStr));
-
-  const [filterData, setFilterData] = useState({
-    likesDirection: '',
-    NFTauthor: 'all',
-    NFTtitle: 'all',
-    NFTdescription: 'all',
-    priceRange: {
-      from: cheapest,
-      to: mostExpensive,
-    },
-  } as FilterType);
-
-  const [isOpen, setIsOpen] = useState(false as boolean);
+import useAppSelector from '../../store/hooks/useAppSelector';
+import {
+  selectNFTsMarketplace,
+  setMarketplaceFilters,
+} from '../../features/marketplace/store/marketplace.slice';
+
+// TODO: Get cheapest and most expensive prices
+const cheapest = 0.00001;
```

Comment:
Hardcode?

Reply:
Are you sure you are reviewing the latest commit?